runtime/src/lib.rs

```diff
@@ -538,6 +543,31 @@ impl pallet_identity::Trait for Runtime {
     type ForceOrigin = EnsureProportionMoreThan<_1, _2, AccountId, CouncilCollective>;
 }
 
+
+parameter_types! {
+    pub const HashId: chainbridge::ResourceId = chainbridge::derive_resource_id(1, &blake2_128(b"hash"));
```

Comment:
I would rename this to cent_nft_hash
```suggestion
    pub const HashId: chainbridge::ResourceId = chainbridge::derive_resource_id(1, &blake2_128(b"cent_nft_hash"));
```